### DISCUSSION TITLE
Icon Size of the map layer elements

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#283] [Q&A] [answered]
How can I separately enlarge the symbols of the map layer elements without needing to enlarge the complete application size over 100%?

**Comment by devemux86** [comment, marked as answer]
It is in "Settings | Map | Appearance | Icon size (map)".

And "Font size (map)" for the labels of map elements.

**Reply by DolomitesTom** [reply]
Thank you

**Reply by Ciutnik** [reply]
By the way, it's worth remembering about the great settings search engine! 🔍 Just click the magnifying glass, type "icon" and the first result found is the one you need: 

<img src="https://github.com/devemux86/cruiser/assets/122647782/28836374-e4cb-4d23-98fb-3713e01ea409" width="50%"/>

This is really a great feature 👍
